zcash_client_sqlite/src/error.rs

```diff
@@ -118,17 +122,22 @@ pub enum SqliteClientError {
     /// A note selection query contained an invalid constant or was otherwise not supported.
     NoteFilterInvalid(NoteFilter),
 
-    /// The proposal cannot be constructed until transactions with previously reserved
-    /// ephemeral address outputs have been mined. The parameters are the account UUID and
-    /// the index that could not safely be reserved.
-    #[cfg(feature = "transparent-inputs")]
-    ReachedGapLimit(AccountUuid, u32),
-
-    /// An ephemeral address would be reused. The parameters are the address in string
-    /// form, and the txid of the earliest transaction in which it is known to have been
-    /// used.
+    /// An address cannot be reserved, or a proposal cannot be constructed until a transaction
+    /// containing outputs belonging to a previously reserved address has been mined. The error
+    /// contains the index that could not safely be reserved.
     #[cfg(feature = "transparent-inputs")]
-    EphemeralAddressReuse(String, TxId),
+    ReachedGapLimit(KeyScope, u32),
```

Comment:
```suggestion
    ReachedGapLimit(TransparentKeyScope, u32),
```

Reply:
This change was accepted as part of 0dcb2b2.